frame/contracts/src/benchmarking/mod.rs

```diff
@@ -47,10 +46,10 @@ use sp_std::prelude::*;
 use wasm_instrument::parity_wasm::elements::{BlockType, BrTableData, Instruction, ValueType};
 
 /// How many batches we do per API benchmark.
-const API_BENCHMARK_BATCHES: u32 = 20;
+const API_BENCHMARK_BATCHES: u32 = 1600;
```

Comment:
Could mention in the comment why these specific values were picked

Reply:
Done.